rdt/hyper_transformer.py

```diff
@@ -346,16 +346,25 @@ def update_transformers(self, column_name_to_transformer):
         if len(self.field_transformers) == 0:
             raise Error(self._DETECT_CONFIG_MESSAGE)
 
+        invalid_columns = list(
+            set(column_name_to_transformer).difference(set(self.field_transformers)))
+        if invalid_columns:
+            raise Error(
+                f'Invalid column names: {invalid_columns}. These columns do not exist in '
+                "the config. Use 'set_config' to write and set your entire config at once."
```

Comment:
This validation and eror message also exists in `update_sdtypes`. Can we pull that out into a function `_validate_column_in_config` or something like that?